checks.d/jenkins.py

```diff
@@ -137,6 +137,16 @@ def _get_build_results(self, instance_key, job_dir):
             self.log.error("Error while working on job %s, exception: %s" % (job_name, e))
 
     def check(self, instance, create_event=True):
+        """
+        DEPRECATED:
+        This Jenkins check is deprecated and not actively developed anymore. It will be
+        removed in a future version of the Datadog Agent. Please move to using the Datadog
+        plugin for Jenkins. More information can be found un the Jenkins Integration panel
+        under the Configuration tab (https://app.datadoghq.com/account/settings#integrations/jenkins)
+        """
+        self.warning("This check is deprecated in favor of our Jenkins Datadog plugin."
+                     " It will be removed in a future version of the Datadog Agent.")
```

Comment:
The link to the integrations panel should suffice imho, since it has details setup instructions
https://github.com/DataDog/dogweb/pull/9584